proposals/threads/Overview.md

```diff
@@ -514,6 +514,10 @@ thrown.
 
 Return `ret` as a Number value.
 
+Note: When [`IsSharedArrayBuffer`][](`M.[[BufferObject]]`) is true, the value `ret`
+should be the result of an atomic read-modify-write of the new size to the
```

Comment:
The only issue here is that a RMW doesn't always return the old value, IIRC (it does in wasm and JS but not in other implementations). Maybe worth clarifying here.

Reply:
Done